### DISCUSSION TITLE
@types/mssql depends on msnodesqlv8 what leads to failing builds due to native compilation step

### DISCUSSION THREAD

**Discussion by NoNameProvided** [DefinitelyTyped/DefinitelyTyped#54287] [Issues with a @types package]
We are using the [mssql](https://www.npmjs.com/package/mssql) package which has its type definitions in [@types/mssql](https://www.npmjs.com/package/@types/mssql). 

The `mssql` packages has two possible driver and uses Tedious as it's default MSSQL driver what is a JS only implementation. However the `@types/mssql` lists the other option the `msnodesqlv8` as a dependency what is a native driver and has a postinstall compilation steps what is failing for us. This change was introduced in the 7.x release line.

I understand that it's included to reuse some type definitions, however I think this should not be the case it's blocks the usage of the type definition for anyone who cannot install the `msnodesqlv8` package.

My suggested solution would be to duplicate the typing for the configuration object for `msnodesqlv8` in the `@types/mssql`  package.

**Comment by ghost** [comment]
https://stackoverflow.com/questions/14772508/npm-failed-to-install-time-with-make-not-found-error

Encountered the same on a Ubuntu VM, and I installed the package as indicated ^ and the postinstall could work.
